making-demos.md

```diff
@@ -171,6 +171,27 @@ TODO
 
 TODO
 
+### Prepending the Firefox DevTools intro video
+
+It's a good idea to also prepend the Firefox DevTools intro video in your new video so it looks more official.
+
+You can download this video [on GitHub's LFS storage here](https://github.com/devtools-html/demos/blob/master/assets/bumper%201080.mov).
+
+You can see an example of it being used at the start of [this video](https://www.youtube.com/watch?v=uZQTEPGdjAg).
+
+#### Mac OS
+
+On Mac OS, you can insert the intro video with Quicktime player by going to the `Edit` menu and selecting the `Add clip to end...` option.
+This will open a dialot for you to pick the intro video. You can then move the inserted clip to before the start of your video by simple drag and drop.
```

Comment:
typo: dialot -> dialog

I'm also not quite sure about the end of the sentence, "by simple drag and drop" sounds weird to me but I'm not sure why or what would be best. Perhaps "by drag and dropping it"?